apps/ledger-live-desktop/src/renderer/components/Web3AppWebview/WalletAPIWebview.tsx

```diff
@@ -79,6 +80,12 @@ function useUiHook(manifest: AppManifest): Partial<UiHook> {
           }),
         );
       },
+      "storage.get": ({ key, storeId }) => {
+        return getStoreValue(key, storeId);
+      },
+      "storage.set": ({ key, value, storeId }) => {
+        setStoreValue(key, value, storeId);
```

Comment:
do we have a more precise spec of what is `storage` and why it is needed for a given web3 app? I'm not super fan of the fact any web3 can query any other sibling web3 app as it doesn't seem to be sandboxed per app (or i missed something?)

Reply:
_"The storage feature will be used for the following use cases:
1. Allow inter-web3app data sharing
2. Allow ledger live to read data stored from web3app (facilitate using tokens in the recover case)
3. In the future, easily "cloud syncing" data from web3app between multiple ledger live clients

It's totally sandboxed from one app to another by default. The sandboxing logic reside in the WalletAPIServer package. For now, we only allow an app to fetch data from its own namespace. In the future, special permissions in the manifest will allow a web3app to fetch data from a namespace it doesn't own."_

c.c. @IAmMorrow

Reply:
ok thanks! this should be in the PR description btw